### ISSUE TITLE
Please tag and release 0.1.2

### ISSUE THREAD

**Issue by innir** [solokeys/solo2-cli#71]
According to the changelog version 0.1.2 was released on 2022-01-07 but you didn't tag nor upload to crates.io - could you please do so?

BTW: I plan to work on packaging solo2-cli for Debian next week :-)

**Comment by nickray** [comment]
Thanks for the reminder, yes it was never released. I changed version to 0.2.0 as it's slightly backwards incompatible, and will release now.

**Comment by nickray** [comment]
A Debian package sounds exciting! It would be great if you can include the shell completions.

**Comment by loveshack** [comment]
> A Debian package sounds exciting! It would be great if you can include the shell completions (+udev rule), and build with the dev-pki feature.

For what it's worth, I looked at packaging for Debian and Fedora and concluded the number of un-packaged dependencies made it quite impractical.

**Comment by innir** [comment]
> For what it's worth, I looked at packaging for Debian and Fedora and concluded the number of un-packaged dependencies made it quite impractical.

Yes, that is unfortunately a rather long (and deep) dependency tree with the whole lpc55 stack missing. I started to package some low hanging fruits now and will continue to do so but I'll wait for my solo2 keys to arrive before I start working on the lpc55 stack ...

**Comment by innir** [comment]
> A Debian package sounds exciting! It would be great if you can include the shell completions (+udev rule), and build with the dev-pki feature.

It would be great if you could add the shell completions to the repository and not only the release. I'd only think about packaging the dev-pki feature if it's not declared experimental anymore to be honest.
